From 5a668d0480d61cd230e27566d0735946915952a5 Mon Sep 17 00:00:00 2001
From: Dan Savilonis <djs@n-cube.org>
Date: Fri, 2 Nov 2012 03:19:03 -0400
Subject: [PATCH] Work around BundleInstall! not working

---
 autoload/vundle/installer.vim | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autoload/vundle/installer.vim b/autoload/vundle/installer.vim
index f7e4375..4d56f94 100644
--- a/autoload/vundle/installer.vim
+++ b/autoload/vundle/installer.vim
@@ -212,7 +212,7 @@ func! s:sync(bang, bundle) abort
 
     if (has('win32') || has('win64'))
       let cmd = substitute(cmd, '^cd ','cd /d ','')  " add /d switch to change drives
-      let cmd = '"'.cmd.'"'                          " enclose in quotes
+      "let cmd = '"'.cmd.'"'                          " enclose in quotes
     endif
 
     let get_current_sha = 'cd '.shellescape(a:bundle.path()).' && git rev-parse HEAD'
-- 
1.7.11.msysgit.1

