From 66d0fb3f24869abf6187ea547cc8b6e864b1a5bc Mon Sep 17 00:00:00 2001
From: Dan Savilonis <djs@n-cube.org>
Date: Mon, 7 May 2012 01:08:13 -0400
Subject: [PATCH] Work around BundleInstall! not working

---
 autoload/vundle/installer.vim |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autoload/vundle/installer.vim b/autoload/vundle/installer.vim
index 5a1a5ee..b1feec8 100644
--- a/autoload/vundle/installer.vim
+++ b/autoload/vundle/installer.vim
@@ -210,7 +210,7 @@ func! s:sync(bang, bundle) abort
 
     if (has('win32') || has('win64'))
       let cmd = substitute(cmd, '^cd ','cd /d ','')  " add /d switch to change drives
-      let cmd = '"'.cmd.'"'                          " enclose in quotes
+      "let cmd = '"'.cmd.'"'                          " enclose in quotes
     endif
   else
     let cmd = 'git clone '.a:bundle.uri.' '.shellescape(a:bundle.path())
-- 
1.7.10.msysgit.1

